ci/Jenkinsfile

```
@@ -109,15 +106,11 @@ pipeline {
 
 def launchLocalTestnet(String name) {
   /* We want to mark job as failed, but save the results. */
-  catchError(
```

You know, you could just set `catchInterruptions: false`:

>`catchInterruptions` : boolean (optional)
>If true, certain types of exceptions that are used to interrupt the flow of execution for Pipelines will be caught and handled by the step. If false, those types of exceptions will be caught and immediately rethrown. Examples of these types of exceptions include those thrown when a build is manually aborted through the UI and those thrown by the timeout step. Defaults to true.

https://www.jenkins.io/doc/pipeline/steps/workflow-basic-steps/#catcherror-catch-error-and-set-build-result-to-failure

---

With `catchInterruptions: false`, the logs won't get saved in case of timeout though.

---

Hmmm, I see, that makes sense. Then if use of `finally` doesn't hide the error that's fine.

---

If it does not then we can handle that by using `updateBuildStatus('FAILURE')` in a `catch`.

---

I have switched it to `post`, this way the syntax is consistent with the other post step for cleaning up workspace

---

Yeah that's cleaner.